src/ceph-disk

```diff
@@ -2191,11 +2353,65 @@ def mount_activate(
 
     path = mount(dev=dev, fstype=fstype, options=mount_options)
 
+    # check if the disk is deactive, change the journal owner, group
+    # mode for correct user and group.
+    if os.path.exists(os.path.join(path, 'deactive')):
+        # logging to syslog will help us easy to know udev triggered failure
+        if not reactivate:
+            unmount(path)
```

Comment:
> Or we could see the log message when we use LOG.info on machine boot stage?

I'm not sure. In any case, such a change belongs to another pull request because you also need to change all the other LOG.info, if that turns out to be necessary. In the context of this pull request, I would keep LOG.info and open a new pull request (or a discussion on the mailing list) regarding the readability of messages at boot time.